src/rpcrawtransaction.cpp

```diff
@@ -808,9 +808,9 @@ UniValue sendrawtransaction(const UniValue& params, bool fHelp)
         throw JSONRPCError(RPC_DESERIALIZATION_ERROR, "TX decode failed");
     uint256 hashTx = tx.GetHash();
 
-    bool fOverrideFees = false;
-    if (params.size() > 1)
-        fOverrideFees = params[1].get_bool();
+    CFeeRate rawMaxFeeRate = CFeeRate(::minRelayTxFee.GetFeePerK() * 10000);
```

Comment:
Exactly! Or probably actually a -maxtxfee rather than feerate - if for some reason you need a very large transaction to send some funds, that's equally likely to constitute a bug that should be investigated manually by a human and fixed.